.github/workflows/check.yml

```diff
@@ -1,60 +1,48 @@
-name: "Check DVC GitHub Action"
-
+name: Check DVC GitHub Action
 on: [push]
```

Comment:
```suggestion
on: [push]

defaults:
  run:
    shell: bash
```

It looks like the Windows tests are failing because it uses PowerShell by default.

Reply:
P.S. `matrix.system` was changed from `windows` to `windows-latest` in #21 which is why this needs updating.